cypress/e2e/awx/resources/hosts.cy.ts

```diff
@@ -37,89 +37,105 @@ describe('host and inventory host', () => {
     cy.visit(`/infrastructure/hosts`);
     createAndEditAndDeleteHost(false, inventory);
   });
-});
 
-function createAndCheckHost(inventory_host: boolean, inventory: string) {
-  const hostName = 'E2E Inventory host ' + randomString(4);
+  it('can work with groups tab', () => {
+    cy.createInventoryHostGroup(organization).then((result) => {
+      const { inventory, host, group } = result;
+      cy.visit('/infrastructure/hosts?page=1&perPage=10&sort=name');
+      cy.contains(host.name || '').click();
```

Comment:
There is missing cy.searchAndDisplayResource(host.name);

It must be done first before doing cy.contains(host.name || '').click();

otherwise with more hosts in the database, you may not see that host in the page and test may fall.

Reply:
this was added